lldb/source/Plugins/Language/CPlusPlus/LibCxxSpan.cpp

```diff
@@ -119,7 +119,7 @@ lldb_private::formatters::LibcxxStdSpanSyntheticFrontEnd::Update() {
     } else if (auto arg =
                    m_backend.GetCompilerType().GetIntegralTemplateArgument(1)) {
 
-      m_num_elements = arg->value.getLimitedValue();
+      m_num_elements = arg->value.GetAPSInt().getLimitedValue();
```

Comment:
Since `Scalar` default initializes both float and integer members, i removed the `isInt` call from prior revisions here